Soccar [Game-Server]/src/nl/soccar/gameserver/controller/socnet/message/ChatMessage.java

```diff
@@ -16,20 +18,42 @@
     private final Privilege privilege;
     private final String message;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)